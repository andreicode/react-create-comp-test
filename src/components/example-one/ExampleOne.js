import React from 'react';
import './style.scss';

const ExampleOne = (props) => {
  return (
    <div className="container">
      <p>ExampleOne111</p>
    </div>
  );
};

export default ExampleOne;
