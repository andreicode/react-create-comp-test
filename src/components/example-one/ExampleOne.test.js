import React from 'react';
import { render } from '@testing-library/react';
import ExampleOne from './ExampleOne';

describe('[COMPONENT] ExampleOne', () => {
  it('renders correctly', () => {
    const { container } = render(<ExampleOne />);
    expect(container).toMatchSnapshot();
  });
});
