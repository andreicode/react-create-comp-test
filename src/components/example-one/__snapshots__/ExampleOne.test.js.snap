// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] ExampleOne renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p>
      ExampleOne111
    </p>
  </div>
</div>
`;
